doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
```

Comment:
We should add some more text that explains what this script does. Also making sure that people understand the risks of running this since it will modify various system parameters with full root access.

Reply:
You are right, I added more text and example usages in it.